jdaviz/configs/imviz/plugins/links_control/links_control.py

```diff
@@ -172,6 +181,12 @@ def _update_link(self, msg={}):
         if self.linking_in_progress:
             return
 
+        if self.need_clear_subsets:
+            raise ValueError("Link type can only be changed after existing subsets "
+                             f"are deleted, but {len(self.app.data_collection.subset_groups)} "
+                             f"subset(s) still exist. To delete them, you can use "
+                             f"`imviz.plugins['Links Control'].delete_subsets()`.")

```

Comment:
`delete_subsets` needs to be added to the list of exposed methods in `user_api`.  I also wonder if just referring to `delete_subsets()` is sufficient since the user arguably already has access to the plugin and they may not have their config named `imviz`.

Reply:
Good call, done in https://github.com/bmorris3/jdaviz/pull/10/commits/86bcc20744e0e56d4645630c90e27a794b6f2e78